### PR TITLE
Auditing workflows for typos and such

### PR DESCRIPTION
**Pull request by ianhodge**

### CODE REVIEW

File: specs/while_loop.yaml [hunk 1]
```diff
@@ -1,14 +0,0 @@
-name: "While loop"
-tags: ["shell"]
-command: "while {{condition}}; do\n\t{{command}}\ndone"
-arguments:
-  -
-    name: "condition"
-    description: "The condition for the while loop"
-  -
-    name: "command"
```
Comment (ianhodge) [inline specs/while_loop.yaml:9]: This was a repeat of another one

Comment (alokedesai) [reply specs/while_loop.yaml:9]: thoughts on deleting all of the top level workflows on top of just this one? I feel like they're already encapsulated within other workflows anyway

Comment (ianhodge) [reply specs/while_loop.yaml:9]: yeah I think that's a good idea

File: specs/kubernetes/list_events_for_a_single_kubernetes_pod.yaml [hunk 2]
```diff
@@ -4,7 +4,13 @@ command: "kubectl get event --namespace {{namespace}} --field-selector involvedO
 tags:
   - kubernetes
 description: Lists the events for a single Kubernetes pod by using the field-selector flag.
-arguments: []
+arguments:
+  - name: namespace
+    description: ~
```
Comment (alokedesai) [inline specs/kubernetes/list_events_for_a_single_kubernetes_pod.yaml:9]: Could we add descriptions here (and any other arguments we added) as well? Will make the UI better

**Review by alokedesai — APPROVED** [review]

**Pull request merged**